packages/config/src/index.ts

```diff
@@ -12,7 +12,9 @@ export const CONVERTKIT_TOKEN = process.env.NEXT_PUBLIC_CONVERTKIT_TOKEN
 export const CONVERTKIT_SIGNUP_FORM =
   process.env.NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM
 export const CONVERTKIT_SUBSCRIBE_API_URL =
-  process.env.NEXT_PUBLIC_SUBSCRIBE_API_URL || '/api/convertkit/subscribe'
+  process.env.NEXT_PUBLIC_CONVERTKIT_SUBSCRIBE_URL ||
+  process.env.NEXT_PUBLIC_SUBSCRIBE_API_URL ||
+  '/api/convertkit/subscribe'
```

Comment:
```suggestion
  process.env.NEXT_PUBLIC_CONVERTKIT_SUBSCRIBE_URL ||
  '/api/convertkit/subscribe'
```

I think that `NEXT_PUBLIC_SUBSCRIBE_API_URL` can be removed since it's not being used anywhere. Looks like a duplicate of `NEXT_PUBLIC_CONVERTKIT_SUBSCRIBE_URL`.